libraries/AP_GPS/AP_GPS.cpp

```diff
@@ -1497,11 +1497,24 @@ void AP_GPS::handle_gps_rtcm_fragment(uint8_t flags, const uint8_t *data, uint8_
 
     const uint8_t fragment = (flags >> 1U) & 0x03;
     const uint8_t sequence = (flags >> 3U) & 0x1F;
+    const bool fragment_consistent = (
+        // the fragment is always consistent with already received ones if
```

Comment:
the suggestion from michael to memcmp is a much better one, then we can drop the option